docs/resources/lts_cross_account_access.md

```diff
@@ -0,0 +1,102 @@
+variable "agency_domain_name" {}
+variable "agency_project_id" {}
+
+resource "huaweicloud_lts_cross_account_access" "test" {
+  name                  = var.name
+  log_agencystream_name = var.agency_stream_name
+  log_agencystream_id   = var.agency_stream_id
+  log_agencygroup_name  = var.agency_group_name
+  log_agencygroup_id    = var.agency_group_id
+  log_stream_name       = var.log_stream_name
+  log_stream_id         = var.log_stream_id
+  log_group_name        = var.log_group_name
+  log_group_id          = var.log_group_id
+  agency_project_id     = var.agency_project_id
+  agency_domain_name    = var.agency_domain_name
+  agency_name           = var.agency_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used.
+  Changing this creates a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the cross account access.
+  Changing this creates a new resource.
+
+* `agency_domain_name` - (Required, String, ForceNew) Specifies the name of the delegator account to verify
+  the delegation. Changing this creates a new resource.
+
+* `agency_name` - (Required, String, ForceNew) Specifies the name of the agency created in IAM by the delegator.
+  Changing this creates a new resource.
+
+* `agency_project_id` - (Required, String, ForceNew) Specifies the delegator project ID.
+  Changing this creates a new resource.
+
+* `log_agencygroup_id` - (Required, String, ForceNew) Specify the log group ID that already exists in the
+  delegated account. Changing this creates a new resource.
+
+* `log_agencygroup_name` - (Required, String, ForceNew) Specify the log group name that already exists in the
+  delegated account. Changing this creates a new resource.
+
+* `log_agencystream_id` - (Required, String, ForceNew) Specifies the log stream ID that already exists in the
+  delegated account. Changing this creates a new resource.
+
+* `log_agencystream_name` - (Required, String, ForceNew) Specifies the log stream name that already exists in the
+  delegated account. Changing this creates a new resource.
+
+* `log_group_id` - (Required, String, ForceNew) Specify the log group ID that already exists in the
+  delegatee account. Changing this creates a new resource.
+
+* `log_group_name` - (Required, String, ForceNew) Specify the log group name that already exists in the
+  delegatee account. Changing this creates a new resource.
+
+* `log_stream_id` - (Required, String, ForceNew) Specifies the log stream ID that already exists in the
+  delegatee account. Changing this creates a new resource.
+
+* `log_stream_name` - (Required, String, ForceNew) Specifies the log stream name that already exists in the
+  delegatee account. Changing this creates a new resource.
+
+* `tags` - (Optional, Map) Specifies the key/value to attach to the cross account access.
```

Comment:
```suggestion
* `tags` - (Optional, Map) Specifies the key/value pairs to associate with the cross account access.
```
Keep consistent with the description for other documents.

Reply:
done